frontend/src/routes/manage/Realm/AddChild.tsx

```diff
@@ -167,7 +167,8 @@ const AddChild: React.FC<Props> = ({ parent }) => {
 
                 <InputWithInfo
                     info={<Trans i18nKey="manage.add-child.path-segment-info">
-                        {{ illegalChars: ILLEGAL_CHARS, reservedChars: RESERVED_CHARS }}
+                        {{ illegalChars: ILLEGAL_CHARS }}
+                        {{ reservedChars: RESERVED_CHARS }}
```

Comment:
Just out of curiosity: Where did you see this warning?

Reply:
In browser console when visiting that page. You can still try it on the main deployment.

```
react_devtools_backend.js:4061 react-i18next:: react-i18next: the passed in object contained more than one variable - the object should look like {{ value, format }} where format is optional. 

{illegalChars: '<>"[\\]^`{|}#%/?', reservedChars: "-+~@_!$&;:.,=*'()"}
```